transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java

```diff
@@ -130,6 +133,20 @@ private static TelephonyManager getTelephonyManager(Context context) {
     return (TelephonyManager) context.getSystemService(Context.TELEPHONY_SERVICE);
   }
 
+  private static int getVersionCode(Context context) {
```

Comment:
For completeness, should we call it packageVersionCode?

Reply:
Sounds good! Fixed.